helmfile/charts/gatekeeper/podsecuritypolicies/templates/default/mutations/run-as-non-root.yaml

```diff
@@ -1,7 +1,20 @@
+{{- $exceptions := concat .Values.mutationExceptions }}
+{{- $namespaces := list }}
+{{- range $namespace, $services := .Values.constraints }}
+{{ $namespaces = append $namespaces $namespace }}
+{{- range $name, $value := $services }}
+{{- if hasKey $value.allow "runAsUser" }}
+{{- if get $value.allow.runAsUser "rule" | eq "RunAsAny" }}
+{{- $exceptions = append $exceptions $value.podSelectorLabels }}
+{{- end }}
+{{- end }}
+{{- end }}
+{{- end }}
```

Comment:
Good to know that it is possible to build objects like this :+1: :eyes:

Reply:
Yes, the power of go templates, though I wish it would be able to short-circuit evaluations since it could have simplified it considerably. :sweat_smile: